api/src/main/java/com/cloud/agent/api/storage/OVFHelper.java

```diff
@@ -37,6 +37,7 @@
 
 import org.apache.cloudstack.utils.security.ParserUtils;
 import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.ObjectUtils;
```

Comment:
Build failing due to this unused import.
```suggestion
```

Reply:
Thanks